src/components/CreateScenarioButton/CreateScenarioButtonHook.js

```diff
@@ -40,8 +43,15 @@ export const useCreateScenarioButton = ({ disabled, onScenarioCreated }) => {
       return [];
     }
 
-    return runTemplates.filter((rt) => runTemplateFilter.includes(rt.id));
-  }, [solution?.data?.runTemplates, workspaceData?.solution?.runTemplateFilter]);
+    const filteredRunTemplate = runTemplates.filter((rt) => runTemplateFilter.includes(rt.id));
+    const translatedRunTemplate = clone(filteredRunTemplate);
```

Comment:
The object runTemplates is directly bind to the state object and so it can be changed by a simple assignment. If we try to do it, the webapp return the error : the object is read only .
I don't have found another way to set the name property by cloning the runTemplates (in this case, the filteredRunTemplate object).